resources/type-system/subtyping.md

```diff
@@ -600,11 +600,20 @@ If last rule applied in D is:
 - QED
 
 
-**Conjecture 1**: `FutureOr<A> <: FutureOr<B>` is derivable iff `A <: B` is
-derivable.
+**Note**: It is not the case that `FutureOr<A> <: FutureOr<B>` is derivable iff
+`A <: B` is derivable.  Counter-example (due to @fishythefish):
 
-Showing that `A <: B => FutureOr<A> <: FutureOr<B>` is easy, but it is not
-immediately clear how to tackle the opposite direction.
+Consider a class `A` which implements `Future<Future<A>>` (such a class is
+definable in Dart).  Let `B` be `Future<A>`.  Note that `A <: B` is not
+derivable, since to show this requires that `Future<Future<A>> <: Future<A>`
+(super-interface), which requires that `Future<A> <: A`, which does not hold.
+
+But we *can* show that `FutureOr<A> <: FutureOr<B>` as follows:
+  - It suffices to show that `A <: FutureOr<B>` and `Future<A> <: FutureOr<B>`
+    - To show that `A <: FutureOr<B>`, it suffices to show that `A <:
+      Future<B>`, which holds by the super-interface rule plus identity.
+    - To show that `Future<A> <: FutureOr<B>` it suffices to show that
+      `Future<A> <: B` which holds by identity.
```

Comment:
Cool! ;-)

But maybe the property is better described by changing line 603 to `**Note:** It is easy to see that `A <: B => FutureOr<A> <: FutureOr<B>`, but the converse does not hold. Counter-example (due to @fishythefish):`.